static/img/api/coordinate-spaces.svg

```diff
@@ -1,262 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why are we removing this file as part of this commit? Did we miss this in the removal of the vision stuff?
If so, please can we cherry-pick its removal onto master so that we can easily restore it if/when we bring back the vision stuff. (I'd be tempted to include something like "missed from removal of vision via `hide-vision` (<hash of hide-vision merge>)" in the commit message so that if we search for the vision removal we also find the new commit.

Reply:
I'm not entirely sure what happened here, I can only assume it's a bad merge on my part. Once this PR merges, it should sort itself out